pyopenms_viz/_core.py

```diff
@@ -1258,6 +1259,35 @@ def _add_box_boundaries(self, annotation_data):
             None
         """
         pass
+    
+    def _compute_3D_annotations(self, annotation_data, x, y, z):
+        def center_of_gravity(x, m):
+            return np.sum(x*m) / np.sum(m)
+        
+        # Contains tuple of coordinates + text + color (x, y, z, t, c)
+        annotations_3d = []
+        for _, feature in annotation_data.iterrows():
+            x0 = feature["leftWidth"]
+            x1 = feature["rightWidth"]
+            y0 = feature["IM_leftWidth"]
+            y1 = feature["IM_rightWidth"]
+            t = feature["name"]
+            c = feature["color"]
```

Comment:
Suggestion not to hard code these column names, include as kwargs for function. Possibly in the future will support more column names.

Reply:
Done!